packages/aws-cdk-lib/core/README.md

```diff
@@ -1629,6 +1629,175 @@ scenarios are (non-exhaustive list):
+```
+
+We also give customers the ability to view all of their applied aspects and override the priority on these aspects.
+The `AspectApplication` class to represents an Aspect that is applied to a node of the construct tree with a priority:
+
+```ts
+/**
+ * Object respresenting an Aspect application. Stores the Aspect, the pointer to the construct it was applied
+ * to, and the priority value of that Aspect.
+ */
+export class AspectApplication {
+  /**
+   * The construct that the Aspect was applied to.
+   */
+  public readonly construct: IConstruct;
+
+  /**
+   * The Aspect that was applied.
+   */
+  public readonly aspect: IAspect;
+
+  /**
+   * The priority value of this Aspect. Must be non-negative integer.
+   */
+  private priority: number;
+}
+```
+
+Users can access AspectApplications on a node by calling `list` from the Aspects class as follows:
+
+```ts
+const app = new App();
+const stack = new MyStack(app, 'MyStack');
+
+Aspects.of(stack).add(new MyAspect());
+
+let aspectApplications: AspectApplication[] = Aspects.of(root).list;
+```
+
+#### Aspects with Third-Party Constructs
+
+When a third-party construct adds and applies its own aspect, we can override that Aspect priority like so:
+
+```ts
+// Import third-party aspect
+import { ThirdPartyConstruct } from 'some-library';
+
+const stack: Stack;
+const construct = new ThirdPartyConstruct(stack, 'third-party-construct');
+
+// Author's aspect - adding to the stack
+const validationAspect = new ValidationAspect();
+Aspects.of(stack).add(validationAspect, { priority: AspectPriority.READONLY } );  // Run later (validation)
+
+// Getting the Aspect from the ThirdPartyConstruct
+const thirdPartyAspectApplication: AspectApplication = Aspects.of(construct).list[0];
+// Overriding the Aspect Priority from the ThirdPartyConstruct to run first
+thirdPartyAspectApplication.priority = 0;
+```
+
+An important thing to note about the `list` function is that it will not return Aspects that are applied to a node by another
+Aspect - these Aspects are only added to the construct tree when `invokeAspects` is called during synthesis.
+
+When using aspects from a library but controlling their application:
+
+```ts
+// Import third-party aspect
+import { SecurityAspect } from 'some-library';
+
+const stack: Stack;
+
+// Application author has full control of ordering
+const securityAspect = new SecurityAspect();
+Aspects.of(stack).add(securityAspect, { priority: 50 } );
+
+// Add own aspects in relation to third-party one
+Aspects.of(stack).add(new MyOtherAspect(), { priority: 75 } );
+```
+
+In all scenarios, application authors can use priority values to ensure their aspects run in the desired order relative to other aspects, whether
+those are their own or from third-party libraries. The standard priority ranges (200 for mutating, 600 default, 1000 for readonly) provide
+guidance while still allowing full flexibility through custom priority values.
+
+### Aspect Stabilization
```

Comment:
> This is controlled by the flag xxx, which is default for new projects (since version x.y.z)

I thought that you'd still need to turn on the feature flag for future versions? How does this work ?

Reply:
NVM, just got it: `defaults: { v2: true }`